VERSION

```diff
@@ -0,0 +1 @@
+1.0.0
```

Comment:
`VERSION` file is not needed if you are not going to make any releases.

Reply:
Will remove